### PR TITLE
Fix sortAlphabetically

### PR DESCRIPTION
**Pull request by micahduron**
#200

- [x] Running `yarn lint` does not trigger any linter errors
- [x] The test of the method you have fixed is passing
- [x] You have written a new skeleton method for someone else to work on!
- [x] You have written tests to accompany your skeleton method

### CODE REVIEW

File: test/rotateArray.test.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+const { rotateArray } = require("../lib");
+
+describe("rotateArray", () => {
+  test("rotateArray([1,2,3], 2) == [2, 3, 1]", () => {
```
Comment (ktilcu) [inline test/rotateArray.test.js]: The test descriptions are more useful as a written statement. something like "Rotates items in array" and "Returns identity for a 1 or 0 length array" "Mutates given array" or "Does not mutate array", etc.

**Comment by ktilcu** [comment]
If you rebase i will merge! Happy Hacktobering!

**Comment by micahduron** [comment]
I think I've addressed everything :)

**Comment by BlakeGuilloud** [comment]
Sweet. Thanks @micahduron !! If you get a chance, we are trying to encourage our contributors to open up an Issue for the skeleton method they have created. Thanks!!

**Pull request merged**